arch/arm64/src/imx9/imx9_lpspi.c

```diff
@@ -1553,6 +1554,10 @@ static void imx9_lpspi_bus_initialize(struct imx9_lpspidev_s *priv)
 {
   uint32_t reg = 0;
 
+  /* Make sure bus is disabled */
+
+  imx9_lpspi_modifyreg32(priv, IMX9_LPSPI_CR_OFFSET, LPSPI_CR_MEN, 0);
```

Comment:
only if priv->refcount == 0, maybe do this below before calling  imx9_lpspi_bus_initialize?